readme.md

```diff
@@ -269,6 +269,20 @@ The following properties are defined:
 * `endpoint`: Base URL for the API, without e.g. the `index/graphql` suffix.
 * `apiVersion`: One of `otp1`, `otp2` or `entur`.
 
+#### TRIAS
+
+*WIP*
+
+The following properties are defined:
+* `endpoint`: Base URL for the API
+* `testingEndpoint`: Base URL for testing purposes
+* `apiVersion`: `1.X` as [available](https://github.com/VDVde/TRIAS/releases)
+* `content-type`: [Content-Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) e.g. `text/xml`
+* `openAPI`: Boolean, whether one need to sign up or not
```

Comment:
I would rename this flag, given that [*OpenAPI*](https://www.openapis.org) is a very well-known spec for describing APIs in a machine-readable way. I propose this:

```suggestion
* `requiresSignUp`: Whether one needs to sign up to obtain an API key.
* `requiresContract` Wether one needs to sign a contract to use the API.
```